internal/workflow/preprocessing_test.go

```diff
@@ -135,6 +137,14 @@ func (s *PreprocessingTestSuite) SetupTest(cfg *config.Configuration) {
 	s.sipPath = sp
 
 	// Register activities.
+	validator, err := bagit_gython.NewBagIt()
+	if err != nil {
+		s.T().Fatalf("initialize bagit: %v", err)
+	}
+	s.env.RegisterActivityWithOptions(
+		bagvalidate.New(validator).Execute,
```

Comment:
You can just use a nil validator here (e.g. `bagvalidate.New(nil).Execute`) and it won't make any difference to the results.  All of the activity calls in these workflow tests are mocked, so the activity is never actually run and the `validator.Validate()` method is never called.

Reply:
@mcantelon you don't even need to use the `bagvalidate.NewNoopValidator()` I've discovered. You can literally just pass a `nil` for the validator:

```go
bagvalidate.New(nil).Execute,
```